logs/logs.go

```diff
@@ -110,6 +113,8 @@ func (l *LogAgent) Run(ctx context.Context) {
 					go l.runSrcToDest(src, dest)
 				}
 			}
+		case <-minuteTicker.C:
```

Comment:
Brought this up in the other comment, but I'm not sure about this edge case: is there strict determinism for when two timers of different frequency should fire at the same time? Do they both fire, does one always take precedence? My concern would be if we never hit this timer chan

Reply:
It is definitely getting hit on my windows 2016 test host.
Every 60th second both tickers will be "runnable".

https://developpaper.com/detailed-explanation-of-select-multiple-selection-in-golang-development/
>Select will randomly execute a runnable case. If there is no case to run, goroutine will block until there is a case to run.

So it is not deterministic which case will run, but as long as the "work" under the 1 second ticker completes in less than a second, the `for` loop wraps around and runs `select` again in which case the 1 minute ticker is the only thing "runnable".

Reply:
Good enough to me